src/biotite/database/uniprot/query.py

```diff
@@ -0,0 +1,201 @@
+# This source code is part of the Biotite package and is distributed
+# under the 3-Clause BSD License. Please see 'LICENSE.rst' for further
+# information.
+
+__name__ = "biotite.database.uniprot"
+__author__ = "Maximilian Greil"
+__all__ = ["SimpleQuery", "CompositeQuery", "search"]
+
+import requests
+import abc
+
+_base_url = "https://www.uniprot.org/uniprot/"
+
+_search_url = ("?query={:}"
+               "&format=list"
+               "&limit={:}")
```

Comment:
This part of the URL can be described more clearly by the `params` parameter of `requests.get()`